curriculum/challenges/english/16-the-odin-project/top-basic-function-projects/top-basic-functions-exercise-a.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 6619240f46cec8e04d77e03a
+title: Basic Functions Exercise A 
+challengeType: 1
+dashedName: top-basic-functions-exercise-a
+---
+
+# --description--
+
+Create a function that takes in an integer. This function should return the given `integer + 7` if the integer is less than `10`. If the integer is greater than or equal to `10`, it should return the given `integer - 3`.
+
+The name of the function should be `addOrSubtract`.
+
+# --hints--
+
+You should have a function called `addOrSubtract`.
+
+```js
+assert.isFunction(addOrSubtract);
+```
+
+Your function should take in an integer as an argument.
+
+```js
+assert.match(addOrSubtract.toString(), /\s*addOrSubtract\(\s*\w+\s*\)/);
+```
```

Comment:
If we are saying that we don't want to check if they created the function at all or created the correct parameter names, then all of these types of tests and hints should be removed.

But IMO, we should at least test that they created a function with the name we are looking for.